test/hotspot/jtreg/vmTestbase/vm/share/gc/HeapOOMEException.java

```diff
@@ -20,18 +20,16 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package metaspace.stressHierarchy.common.exceptions;
+package vm.share.gc;
 
 /**
- * Usually this means that we got OOME:heap while trying to gain OOME:metaspace.
- * We pass test in this case as this breaks test logic. We have dedicated test configurations
- * for OOME:heap provoking class unloading, that why we are not missing test coverage here.
+ * This class is used to differ OOME in metaspace and heap when trigger class unloading.
```

Comment:
That doesn't scan so well.  Perhaps something like
"This class is used to distinguish between OOME in metaspace and OOME in heap when triggering class unloading."